requirements.txt

```
@@ -1,2 +1,3 @@
 pyzbar
 pdf2image
+openupgradelib @ git+https://github.com/OCA/openupgradelib.git
```

I don't think this format is correct... At least I never saw it. I think it's just:

```suggestion
git+https://github.com/OCA/openupgradelib.git
```

---

@Yajo This format is the right one to declare a python dependency installed from a git repository into a requirements.txt file

---

OK, TIL. :smile: